apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClient.cs

```diff
@@ -167,6 +132,29 @@ public UrlSigner UrlSigner
                 UnauthenticatedAccess = true
             }.Build();
 
+        /// <summary>
+        /// Creates a URL signer built base on this client, that is using the same credential as this client, as well
+        /// as defaulting to this client's URI scheme, host and port. If the credential used by this client is not compatible
+        /// with <see cref="UrlSigner"/> this method will throw <see cref="InvalidOperationException"/>.
+        /// See <see cref="UrlSigner"/>'s documentation for more information on compatible credentials.
+        /// </summary>
+        /// <remarks>
+        /// Because credentials used by this client may changed, this method will always return a new <see cref="UrlSigner"/> instance
```

Comment:
I think we can probably remove these remarks now - I'd expect a method called "Create" to return a new instance anyway.